src/java.base/share/classes/jdk/internal/reflect/ReflectionFactory.java

```diff
@@ -39,9 +40,13 @@
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 import java.lang.reflect.Modifier;
+import java.lang.reflect.Proxy;
 import java.security.PrivilegedAction;
```

Comment:
Unneeded import.

Reply:
This is used down below on line 423:

```java
    private static boolean hasDefaultOrNoSerialization(Class<?> cl) {
        return ! Serializable.class.isAssignableFrom(cl)
            || cl.isInterface()
            || cl.isArray()
            || Proxy.isProxyClass(cl) // <=== right here
            || Externalizable.class.isAssignableFrom(cl)
            || cl.isEnum()
            || cl.isRecord()
            || cl.isHidden()
            || nonSerializableLeafClasses.contains(cl);
    }
```

Reply:
Sorry, I meant to flag PrivilegedAction.

Reply:
Ah, I see. It looks like it's still used though, in `GetReflectionFactoryAction` which hasn't been removed yet. I guess it'll be removed in one of the sub-tasks of [JDK-8338411](https://bugs.openjdk.org/browse/JDK-8338411)?

Reply:
ok, as is.